### PR TITLE
Custom widgets

### PR DESCRIPTION
**Pull request by divyeshp07**
widgets moved to core

### CODE REVIEW

File: lib/core/widgets/main_btn_widget.dart [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
```
Comment (Sireen-Official) [inline lib/core/widgets/main_btn_widget.dart:1]: Optimizations required

File: lib/core/widgets/product_card_widget.dart [hunk 2]
```diff
@@ -0,0 +1,144 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:rent_hub/core/theme/app_theme.dart';
+
+class ProductCardWidget extends StatelessWidget {
+  final String productName;
+  final double price;
+  final String productLocation;
+  final double destance;
```
Comment (Sireen-Official) [inline lib/core/widgets/product_card_widget.dart:10]: Spell correction

File: lib/core/widgets/product_card_widget.dart [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    required this.price,
+    required this.productLocation,
+    required this.destance,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final apptheme = AppTheme.of(context);
+
+    return InkWell(
```
Comment (Sireen-Official) [inline lib/core/widgets/product_card_widget.dart:24]: Need border radius

**Review by Sireen-Official — CHANGES_REQUESTED** [review]

File: lib/core/widgets/main_btn_widget.dart
Comment (thadathilsinan) [inline lib/core/widgets/main_btn_widget.dart:1]: Use spaces class instead of hardcoded values

**Review by thadathilsinan — CHANGES_REQUESTED** [review]

**Review by Sireen-Official — APPROVED** [review]

**Review by thadathilsinan — APPROVED** [review]

**Pull request merged**